### DISCUSSION TITLE
RTX4090D被国内魔改双倍显存

### DISCUSSION THREAD

**Discussion by Myitian** [share121/inter-knot#1810] [其他]
![cover](https://cdn.videocardz.com/1/2024/08/RTX4090D-48GB-HERO-2048x1065.jpg)
据称最近国内有云服务提供商向客户租赁 RTX4090D 48G 和 RTX4080 32G 显卡。
这些定制显卡更适合用于训练大语言模型和其他生成式 AI 模型，这需要大量的计算能力和内存容量。

---
来源：
[【推特】青龍聖者@bdsqlsz](https://x.com/bdsqlsz/status/1821838464108917123)
[【VideoCardz】RTX 4090D and RTX 4080 SUPER get doubled memory](https://videocardz.com/newz/nvidia-geforce-rtx-4090d-with-48gb-and-rtx-4080-super-32gb-now-offered-in-china-for-cloud-computing)
[【Bilibili】BV1UW421X7gF 显卡日报8月12日｜RTX4090D被国内厂家成功魔改 显存翻倍](https://b23.tv/BV1UW421X7gF)

**Comment by cpu644** [comment]
有没有gt610魔改 我想把我的gt610 10gb改成20（狗头
![image](https://github.com/user-attachments/assets/b75dd0b9-490a-41de-bfeb-9b579d6939f2)

**Comment by Rain-YYC** [comment]
不是，啊？

**Comment by Rain-YYC** [comment]
给Fairy用岂不是让Fairy更智能？
